conn_test.go

```diff
@@ -192,7 +218,9 @@ var _ = Describe("Connection", func() {
 
 		// now allow the address and make sure the connection goes through
 		clientTransport.(*transport).clientConfig.HandshakeTimeout = 2 * time.Second
-		filters.AddFilter(ipNet, filter.ActionAccept)
+		cg.lk.Lock()
+		cg.allowAll = true
```

Comment:
A test for `InterceptSecured` would be great (mostly for regression testing).

Reply:
This is done.